Dockerfile

```
@@ -0,0 +1,26 @@
+ARG PYTHON_VERSION=3.7
+
+FROM python:${PYTHON_VERSION}
+
+RUN apt-get update \
+    && apt-get -y install openmpi-bin libopenmpi-dev \
+    && apt-get -y install swig  \
```

Since we plan to remove `swig` from Optuna dependencies by https://github.com/optuna/optuna/pull/1106,  how about adding a TODO comment as follows:
```suggestion
    && apt-get -y install swig  \  # TODO: Remove this line once https://github.com/optuna/optuna/pull/1106 is merged.
```